cmd/routed-eni-cni-plugin/driver/mocks/driver_mocks.go

```diff
@@ -1,4 +1,4 @@
-// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
`// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.` is the correct header. We should remove all years.